bin/genhtml

```diff
@@ -4145,16 +4145,15 @@ sub format_count($$)
 
 #
 # write_source_line(filehandle, line_num, source, hit_count, converted,
-#                   brdata, add_anchor)
+#                   brdata)
```

Comment:
You also need to update the subroutine prototype earlier in the source file. In its current form, the change results in warnings:
```
Prototype mismatch: sub main::write_source_line (*$$$$$$) vs (*$$$$$) at genhtml line 4228.
```

Reply:
Done